pkg/process/checks/net_test.go

```diff
@@ -55,7 +55,10 @@ func TestDNSNameEncoding(t *testing.T) {
 		"1.1.2.4": {Names: []string{"host4.domain.com"}},
 		"1.1.2.5": {Names: nil},
 	}
-	ex := parser.NewServiceExtractor(ddconfig.MockSystemProbe(t))
+	conf := ddconfig.MockSystemProbe(t)
+	enabled := conf.GetBool("system_probe_config.process_service_inference.enabled")
+	useWindowsServiceName := conf.GetBool("system_probe_config.process_service_inference.use_windows_service_name")
+	ex := parser.NewServiceExtractor(enabled, useWindowsServiceName)
```

Comment:
Just like in `service_test.go`, you don't need to use a mock here or in any other test using the service extractor. The constructor just accepts boolean values which you can use directly.

Reply:
specifically in `net.go`, arent some of the tests also testing if it correctly sources the value from the system-probe? Like the one in line 627?

Reply:
To clarify, you shouldn't need to use a mock config **specifically** for getting the service extractor values anymore. If the mock config is used for other purposes in the test you should keep it.

Reply:
ok, think i addressed this now